### ISSUE TITLE
JS API doesnt set Id

### ISSUE THREAD

**Issue by boldtrn** [nickpeihl/leaflet-sidebar-v2#27]
If you create the sidebar via the js api and specify a container id, the id is not set to the HTML element.

This could be fixed by something like:
```
if (!container){
      container = L.DomUtil.create('div', 'leaflet-sidebar collapsed');
      container.id = this.options.container;
}
```

**Comment by noerw** [comment]
Good suggestion, will include it in the next release!
I should have time for that within the next month, if you need it earlier please submit a pull (maybe also with your other suggestions).

**Comment by boldtrn** [comment]
Thanks for the reply. This only works if you use html though. If you use the JS API this does not work.

**Comment by boldtrn** [comment]
No need to hurry, I forked this repo already and fixed this. I required a couple of breaking changes and I was a bit in a hurry :), so I only created issues here.

You can have a look at the changes [here](https://github.com/nickpeihl/leaflet-sidebar-v2/compare/master...boldtrn:master), feel free to cherry pick a change or let me know if I should add a PR for any of the changes.

**Comment by noerw** [comment]
fixed in just released v3.0.5
